### PR TITLE
make some dashboard components reusable, add FlowRunHistoryCard

### PR DESCRIPTION
**Pull request by brandonreid**
This makes some of the dashboard cards generic and adds the FlowRunHistoryCard component.

A nebula PR is ready, I'll make a OSS PR along with it once this is approved.

Here's what it looks like in Nebula local:
image

Feedback on some of the naming conventions would be great!

### CODE REVIEW

File: src/components/SimpleTimeSpanFilter.vue
Comment (pleek91) [inline src/components/SimpleTimeSpanFilter.vue:1]: nit: I don't think the prefix simple adds and value here and I would remove it.

File: src/components/WorkspaceDashboardFlowRunsCard.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 <script lang="ts" setup>
   import { computed } from 'vue'
-  import DashboardStatistic from '@/components/DashboardStatistic.vue'
+  import ValueKeyStatistic from '@/components/ValueKeyStatistic.vue'
```
Comment (pleek91) [inline src/components/WorkspaceDashboardFlowRunsCard.vue:18]: linting warning

File: src/types/flow.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export type FlowStatsFilter = {
```
Comment (pleek91) [inline src/types/flow.ts:1]: Trying to figure out where this is used outside of the mappers. I'm assuming this is used at implementation on the actual flow page?

Comment (brandonreid) [reply src/types/flow.ts:1]: It's only used in mappers, but in nebula there's an additional `mapFlowStatsFilterTo**EventsFilter**` that uses it

Comment (pleek91) [reply src/types/flow.ts:1]: Yep, makes sense 👍

**Review by pleek91 — APPROVED** [review]

**Pull request merged**